test/integration/flags.go

```diff
@@ -33,7 +33,7 @@ func TestMain(m *testing.M) {
 
 var binaryPath = flag.String("binary", fmt.Sprintf("../../out/%s-amd64/minishift", runtime.GOOS), "path to minishift binary")
 
-var vmDriver = getenv("VM_DRIVER", "virtualbox") // Get the driver value from environment variable or use default
+var vmDriver = getenv("MINISHIFT_VM_DRIVER", "virtualbox") // Get the driver value from environment variable or use default
```

Comment:
!? - You are changing nothing here, expect renaming the variable. That's not the intention here. If it were just about the name, we might as well skip it. 

The whole idea is that if you use 'MINISHIFT_VM_DRIVER', you don't have to use the '--vm-driver flag' at all, since Viper will take the setting from the environment https://github.com/minishift/minishift/blob/master/docs/using.md#environment-variables

So just making sure that the variable is set in the environment which starts the test will make it work.